tests/Feature/FileAdder/IntegrationTest.php

```diff
@@ -594,3 +596,22 @@
 
     expect($result->getStatusCode())->toEqual(200);
 });
+
+it('will throw and exception and not create a record in database if file cannot be added', function () {
+
+    $this->expectException(DiskCannotBeAccessed::class);
+
+    config()->set('filesystems.disks.minio_disk', [
```

Comment:
Is this a deliberately wrongly configured disk?

Maybe name it `invalid_disk` for clarity.

Reply:
Done, nice catch.